### PR TITLE
refactor(utxo-lib): add address tests, fixtures

### PR DESCRIPTION
**Pull request by OttoAllmendinger**
We will have to change some address formatting/parsing code and should have
some tests ready for that.

Issue: BG-32852

204e4044b
fix(utxo-lib): fix imports in test

Concurrent merge oopsie

Issue: BG-32852

### CODE REVIEW

File: modules/utxo-lib/test/address/address.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          .reduce((all: AddressTestVector[], v) => [...all, ...v], []);
+
+        refVectors = await readFixture(network, vectors);
+      });
+
+      it('matches test vectors, parses to scriptPubKeyHex', function () {
+        assert.strictEqual(vectors.length, refVectors.length);
+        vectors.forEach((v, i) => {
+          assert.deepStrictEqual(v, refVectors[i]);
+          const [scriptType, scriptPubKeyHex, address] = v;
```
Comment (sangaman) [inline modules/utxo-lib/test/address/address.ts:61]: Is `scriptType` deliberately unused here?

**Review by sangaman — APPROVED** [review]
Very nice test suite, will be useful in ensuring the functionality isn't impacted by future refactoring.

File: modules/utxo-lib/test/address/address.ts (hunk 1 above)
Comment (OttoAllmendinger) [reply modules/utxo-lib/test/address/address.ts:61]: yes

I could also write this as `[, scriptPubKeyHex, address] = v;` if you think it is cleaner

Comment (sangaman) [reply modules/utxo-lib/test/address/address.ts:61]: It's not a big deal if you're not going to change the PR for other reasons. Mainly wanted to make sure that something wasn't being overlooked.

**Pull request merged**